moved/src/state_actor.rs

```diff
@@ -193,9 +193,10 @@ impl<
                 response_channel,
                 include_transactions,
             } => response_channel.send(match height {
-                BlockNumberOrTag::Number(height) => self.block_queries.by_height(&self.block_memory, height, include_transactions),
-                // TODO: Support block "tag"
-                _ => None,
+                Number(height) => self.block_queries.by_height(&self.block_memory, height, include_transactions),
+                Latest | Finalized | Safe => self.block_queries.by_height(&self.block_memory, self.height, include_transactions),
+                Earliest => self.block_queries.by_height(&self.block_memory, 0, include_transactions),
+                Pending => None,
```

Comment:
I don't know we should support this, but my understanding was `Pending` meant the transactions in the `mem_pool`.